mandatory/3-function-output.js

```diff
@@ -1,14 +1,22 @@
 // Add comments to explain what this function does. You're meant to use Google!
+// this function is going multiply numbers .
+
+// answer The Math.random() function returns a floating-point, pseudo-random number in the range 0 to less than 1 
+// (inclusive of 0, but not 1) with approximately uniform distribution over that range — which you can then scale to your desired range. The implementation selects the initial seed to the random number generation algorithm; 
+// it cannot be chosen or reset by the user.
 function getRandomNumber() {
   return Math.random() * 10;
 }
+// math.round will round a given number to the nearest integer.
```

Comment:
Great job Ansuj. Here is the 'Math.random' not 'Math.round'. just if you want to change its explanation.